cmd/sops/main.go

```diff
@@ -351,6 +360,10 @@ func main() {
 							Name:  "azure-kv",
 							Usage: "the Azure Key Vault key URL the new group should contain. Can be specified more than once",
 						},
+						cli.StringSliceFlag{
+							Name:  "hc-vault",
```

Comment:
This and all further references should be `hc-vault-transit`, no?

Reply:
Yes, that is correct

Reply:
Can you change it please? It still shows `hc-vault`.